packages/adapters-library/src/core/apy-calculators/helpers.ts

```diff
@@ -65,6 +66,9 @@ export const createApyCalculatorFor = async (
     if (protocolToken.underlyingTokens.length === 1)
       return new BalanceOfApyCalculator()
 
+    if (protocolToken.underlyingTokens.length === 2)
+      return new ConstantProductAmmApyCalculator()
```

Comment:
Here's where we assume the formula for Constant Product AMMs can be generalized to any pool with 2 underlying tokens